### PR TITLE
Proposing changes to Engineering's Accessibility Page

### PR DESCRIPTION
**Pull request by mgifford**
I think this is a better description. Certainly, there needs to be more overlap between the Accessibility Practice area and the Engineering page that describes it. 

Initially I just wanted to point to a better link on d.o, but got sucked into it. 

----
:books: Documentation preview :books:: https://civicactions-handbook--1231.org.readthedocs.build/en/1231/

### CODE REVIEW

File: practice-areas/engineering/accessibility.md [hunk 1]
```diff
@@ -4,22 +4,22 @@ title: Accessibility
 
 # Accessibility
 
-We implement Section 508 standards and WCAG compliant websites so that people with all types of disabilities, whether it be physical, mental, or visual impairment, have access to our sites.
+We implement Section 508 compliant sites and strive to meet the latest Web Content Accessibility Guidelines (WCAG). We do this so that everyone can access our sites, even if they have a permanent, temporary or situtational disability. The Engineering team is very aware of the advantages of open standards, and semantic code for building future-compatible applications. CivicActions has an [Accessibility Practice Area](/en/latest/practice-areas/accessibility/), but is also a component of all other practice areas at CivicActions.
```
Comment (dmundra) [inline practice-areas/engineering/accessibility.md]: @mgifford can you change the path to 'Accessibility Practice Area' to `../accessibility/README.md`.

File: practice-areas/engineering/accessibility.md [hunk 2]
```diff
@@ -4,22 +4,22 @@ title: Accessibility
...
--   We use good practices in using semantic markup in page templates and report design to define content hierarchy (e.g. primary head is h1, secondary head is h2, etc.).
+-   We test sites using a combination of automated and manual testing. We leverage automated testing in the browser, with site-wide scans and also through integration in our CI/CD pipeline.
+-   We follow the best practices of the Drupal community, which includes accessibility.
 
 ## When we do this
 
 -   We strive to produce work that is accessible to people of all abilities, regardless of client. However, we recognize that the level of accessibility compliance and prioritization can be influenced by budgetary and contractual implications.
 -   We aim to do accessibility work continuously, as part of our agile process. Accessibility scans should be performed on a per-ticket basis and signed off on before work is considered complete.
--   Accessibility should NOT BE left until the end of a project, but baked in throughout its lifespan.
+-   We know that accessibility cannot NOT BE left until the end of a project, but baked in throughout its lifespan.
```
Comment (jastraat) [inline practice-areas/engineering/accessibility.md]: We know that accessibility CANNOT be left

File: practice-areas/engineering/accessibility.md (hunk 1 above)
Comment (mgifford) [reply practice-areas/engineering/accessibility.md]: Yup.

File: practice-areas/engineering/accessibility.md (hunk 2 above)
Comment (mgifford) [reply practice-areas/engineering/accessibility.md]: Ok. For all of you who don't enjoy double negatives and inconsistent case. :)  
  
Thanks @jastraat

**Review by dmundra — APPROVED** [review]
Looks good. Thanks @mgifford and @ContentBrewmaster.

**Pull request merged**